config.go

```diff
@@ -279,9 +279,9 @@ type Config struct {
 	AutomaticCutover bool
 
 	// This specifies whether or not Ferry.Run will handle SIGINT and SIGTERM
-	// by dumping the current state to stdout.
-	// This state can be used to resume Ghostferry.
-	DumpStateToStdoutOnSignal bool
+	// by dumping the current state to stdout and the error HTTP callback.
+	// The dumped state can be used to resume Ghostferry.
+	DumpStateOnSignal bool
```

Comment:
It still feels like dumping to stdout is all wrong. 

This will spam logs for the mover and we will get asked about it if there's a large number of moves being interrupted all the time.

Don't have to block but be forewarned 🙂

Reply:
For this concern we can just redirect stdout to /dev/null if needed. stderr contains all the useful logs.